test/functional/page_objects/console_page.ts

```diff
@@ -79,6 +81,35 @@ export function ConsolePageProvider({ getService }: FtrProviderContext) {
     public async getRequestFontSize() {
       return await this.getFontSize(await this.getRequestEditor());
     }
+
+    public async dismissTutorial() {
+      try {
+        const closeButton = await find.byCssSelector('.euiFlyout__closeButton');
+        await closeButton.click();
+      } catch (e) {
+        // Ignore because it is probably not there.
+      }
+    }
+
+    public async appendTextToEditor(editor: WebElementWrapper, browser: any, text: string) {
+      // This focusses the cursor on the bottom of the text area
+      const content = await editor.findByCssSelector('.ace_content');
+      await content.click();
+      const textArea = await editor.findByCssSelector('#ConAppInputTextarea');
+      for (const key of text.split('')) {
+        await textArea.pressKeys(key);
+        // This is needed so that the browser has time to look up each character for autocomplete
+        await PageObjects.common.sleep(100);
```

Comment:
I am a bit hesitant to put hard coded sleeps in a test especially with the latency that's usually in CI. Basically, if CI is running slow the test will fail. This can definitely be flaky. We have a function in the retry service that allows you to wait for a condition. so we should probably use that. I think it's `retry.waitFor()`. Check that function out and I think it will achieve the same thing using our own internal wait mechanisms.